pkg/test/ginkgo/options_monitor_events.go

```diff
@@ -113,6 +117,60 @@ func (o *MonitorEventsOptions) End(ctx context.Context, restConfig *rest.Config,
 	o.endTime = &t
 	o.recordedResources = o.monitor.CurrentResourceState()
 
+	removeNTimes := regexp.MustCompile(`\s+\(\d+ times\)`)
```

Comment:
Lets move all this code you're adding to a function so you can control inputs and write a nice unit test for it. something like 

```
missedPathologicalEvents = fillInMissedPathologicalEvents(events)
events = append(events, missedPathologicalEvents)
```

Reply:
this was a very good suggestion.  I can test far faster this way.  Thanks much!